executor/src/main/java/com/redhat/service/bridge/executor/Executor.java

```diff
@@ -15,14 +20,30 @@
 
     private final ProcessorDTO processor;
 
-    public Executor(ProcessorDTO processor) {
+    private final Set<String> templates;
+
+    private final FilterEvaluator filterEvaluator;
+
+    private final TemplateFactory templateFactory;
+
+    public Executor(ProcessorDTO processor, TemplateFactory templateFactory, FilterEvaluator filterEvaluator) {
         this.processor = processor;
+        this.templates = buildTemplates(processor);
+        this.filterEvaluator = filterEvaluator;
+        this.templateFactory = templateFactory;
     }
 
     public void onEvent(CloudEvent cloudEvent) {
-        LOG.info("Received event with id '{}' for Processor with name '{}' on Bridge '{}", cloudEvent.getId(), processor.getName(), processor.getBridge().getId());
+        LOG.info("[executor] Received event with id '{}' for Processor with name '{}' on Bridge '{}", cloudEvent.getId(), processor.getName(), processor.getBridge().getId());
 
-        //TODO - consider if the CloudEvent needs cleaning up from our extensions before it is handled by Actions
+        if (matchFilters(cloudEvent)) {
+            LOG.info("[executor] Filters of processor '{}' matched for event with id '{}'", processor.getId(), cloudEvent.getId());
+            // TODO - CALL ACTIONS;
+            // TODO - consider if the CloudEvent needs cleaning up from our extensions before it is handled by Actions
+            return;
+        }
+        LOG.debug("[executor] Filters of processor '{}' did not match for event with id '{}'", processor.getId(), cloudEvent.getId());
+        // DO NOTHING;
```

Comment:
I think an `if`/`else` without the `return` is easier to read
```suggestion
        if (matchFilters(cloudEvent)) {
            LOG.info("[executor] Filters of processor '{}' matched for event with id '{}'", processor.getId(), cloudEvent.getId());
            // TODO - CALL ACTIONS;
            // TODO - consider if the CloudEvent needs cleaning up from our extensions before it is handled by Actions
        }
        else {
            LOG.debug("[executor] Filters of processor '{}' did not match for event with id '{}'", processor.getId(), cloudEvent.getId());
            // DO NOTHING;
        }
```